src/packets.toit

```diff
@@ -5,12 +5,38 @@
 import bytes
 import binary
 import reader
+import .client show Client  // For toitdoc reference.
 
 import .topic_filter
 
 interface PacketIDAck:
   packet_id -> int
 
+class LastWillConfig:
```

Comment:
I wonder if having Config as part of this name adds anything. I'd suggest removing it.

Reply:
removed.
done.